### PR TITLE
Added MLOps.NET.Azure.IntegrationTests and running them with Azurite

### LINKED ISSUE #28: Add integration tests for Azure Storage Provider
We would need to add integration tests to ensure that we can properly set up
- Experiments
- Runs
- Metrics
- Log models

The connection string could be kept as part of repo secrets

### LINKED ISSUE #134: Add SQL Server Integration Tests to CI pipeline
**Is your feature request related to a problem? Please describe.**
We currently can't run our SQL server tests in our CI pipeline as it requires SQL server.

**Describe the solution you'd like**
It may make sense to run these tests in a Docker container of sorts, or against a container with SQL Server installed on it.

### PR DESCRIPTION
**Pull request by aslotte**
## Resolves
Fixes #134 
Fixes #28 

## Description
Added a simple integration test project for Azure and started running it with an Azurite service container in GitHub

### CODE REVIEW

**Comment by aslotte** [comment]
This is ready for a review! 🥇

File: src/MLOps.NET.Azure.IntegrationTests/StorageAccountModelRepositoryTests.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    [TestCategory("IntegrationTestsAzure")]
+    [TestClass]
+    public class StorageAccountModelRepositoryTests
+    {
+        private StorageAccountModelRepository sut;
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            sut = new StorageAccountModelRepository("DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;");
```
Comment (AnoojNair) [inline src/MLOps.NET.Azure.IntegrationTests/StorageAccountModelRepositoryTests.cs:19]: @aslotte , lets move this to config, a bit more cleaner.

File: src/MLOps.NET.Azure.IntegrationTests/MLOps.NET.Azure.IntegrationTests.csproj [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <NoWarn>1591</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="5.10.3" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.1.0" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.1.0" />
+    <PackageReference Include="coverlet.collector" Version="1.2.0" />
+    <PackageReference Include="Moq" Version="4.14.3" />
```
Comment (AnoojNair) [inline src/MLOps.NET.Azure.IntegrationTests/MLOps.NET.Azure.IntegrationTests.csproj:17]: do we need the moq package here?

**Comment by AnoojNair** [comment]
@aslotte , this looks really cool.. :)

**Comment by aslotte** [comment]
@AnoojNair good points! I added a `MLOps.NET.Tests.Common` project in which we can isolate configuration logic and other items going forward (e.g. we may want to keep data for training and other common things there)

**Pull request merged**